smoke-test/src/main/java/org/opennms/smoketest/utils/RestClient.java

```diff
@@ -422,7 +422,7 @@ public void triggerBackup(final String requestDTO) {
     }
 
     public JsonNode getBackups() throws IOException {
-        final var result = getBuilder(getTarget().path("device-config"))
+        final var result = getBuilder(getTarget().path("device-config").queryParam("limit", "0"))
```

Comment:
Ah, common. Give it a newline.

Reply:
Here I"m not sure it needs a new line as it is part of the constructor and the next 2 lines are not. It makes more sense to have it on a single line rather than split and confuse it with the following lines.